projects/README.md

```diff
@@ -37,14 +37,22 @@
 	Build with:
 		make PLATFORM=BITTBOY
 
-##### MIYOOMINI: Compile under Linux
-	Required libs:
-		sudo apt install -y libncurses-dev
-	Dev kit:
-		https://github.com/djdiskmachine/miyoomini-toolchain-buildroot/releases/download/1.0.0/miyoomini-toolchain.tar.xz
-	    install in /opt/
-	Build with:
-		make PLATFORM=MIYOOM
+##### MIYOO: Compile with containerized Sdk
+
+For the Miyoo Mini the recommended way to build is using a community Sdk in docker.
+
+<https://github.com/OnionUI/dev-miyoomini-toolchain>
+
+TBD: document building customSDL
```

Comment:
Can we consider prepackaging the dependencies for Little Piggy Tracker? My experience is that these buildroot - based libs go stale quite quickly.

Reply:
yeah, if you can set up or recommend a repository with releases to subscribe to I can add that to the post build script. I'm still using a ripped library on my test unit.

Reply:
Where can I find this ripped lib?